### ISSUE TITLE
Xenko launcher doesn't work

### ISSUE THREAD

**Issue by BartjeD** [xenko3d/xenko#569]
**Release Type**: Official release / lancher 3.0.8

**Version**: Xenko 3.1 / launcher 3.0.8

**Platform(s)**: Windows 10

**Describe the bug**
Once Xenko has been installed the launcher doesn't allow any options and any actions other than closing it or clicking on some links in the bottom left card.

**To Reproduce**
Install Xenko and open the launcher

**Expected behavior**
Being able to add a new project or do something to integrate with VS 2019.
VS 2019 and VS 2019 preview are both installed.

**Screenshots**
![image](https://user-images.githubusercontent.com/49863641/70861879-88acae00-1f34-11ea-9241-a8240afd63a9.png)

**Log and callstacks**
The event viewer does not contain any stacktraces.

**Comment by BartjeD** [comment]
By some magical process the launcher has suddenly upgraded to version 3.09 and is now showing options.

There was apparantly a wait time of about 5 mintues before anything happened.

It would be a lot more user friendly to show the users that `something` is happening while the laucher is upgrading itself or veryifing that it's the latest version. Or something. 

Because now I filed an error report apparantly for nothing.

**Comment by BartjeD** [comment]
I'd like to add that this issue almost made me reconsider trying out Xenko.
It felt very immature that something appeared to go wrong with the installation process.

The fact that it actually went right but needed some time for a background porcess, I'm assuming, is positive. But there's a risk of losing new users if this process isn't visible with a progress bar or something like that.

**Comment by xen2** [comment]
Thanks for the report.

Please note that this is not the typical/expected behavior:
1. Latest launcher is 3.0.9 but current one in `XenkoSetup.exe` is 3.0.8 which then auto-upgrade (we should upgrade `XenkoSetup.exe`, but usually it's not supposed to be bothering, cf next point)
2. As soon as launcher starts, it querys the various nuget feeds to check if a new version is available. This is supposedly very fast if there is normal internet connectivity (it's just a few HTTPS requests).
3. If there is any new version, it should then display the self-update popup and then start download (5 mb). So the download time should not affect the popup display.

I am not sure as to why it would take 5 minutes to do this small HTTPS check from step 2.
Could it be that your internet was unstable? Or that you have some failing nuget feeds that blocks everything else?

Note that we don't block UI until this quick check is done because we want Xenko to be usable even without internet connectivity.

Anyway, I will soon release an updated `XenkoSetup.exe`.

**Comment by BartjeD** [comment]
Hi Xen2, 

First of all, thanks for replying to this topic even though I closed it - I think its very professional that you did so. Gives me a very postiive impression. :)

My internet connection was stable and working at the time. I verified this because I noticed succesful NuGet requests after restarting and using fiddler to determine if the launcher was doing anthing.

I also noticed it quering a NuGet repository that I typically only use for work. And which is unvailable unless I start my work VPN. I didn't think anything of it at the time because the other requests appeared to be succeeding.

Could it be the case that it was trying to query an unreachable repository?
This repository is listed at the bottom of my NuGet feeds in Visual Studio. It typically shouldn't be used. So I'm not sure why it was being queried.
